packages/connect/src/index.d.ts

```diff
@@ -28,9 +28,13 @@ type Store = {
   actions?: Actions;
 };
 
+type Primitive = number | string | boolean | symbol | undefined | null;
```

Comment:
You will notice that this type is duplicated in the [https://github.com/frontity/frontity/pull/610/files#diff-936fe7efe4a257e580cf24c0a2084d2f1428e37b542edac7b6db3f932c3514d6R57](packages/types/src/utils.ts).

This should be cleaned in the PR when we fork react-easy-state.